devito/passes/clusters/factorization.py

```diff
@@ -0,0 +1,139 @@
+from sympy import Add, Mul, collect, collect_const
+
+from devito.passes.clusters.utils import dse_pass
+from devito.symbolics import estimate_cost, retrieve_scalars
+from devito.tools import ReducerMap
+
+__all__ = ['factorize']
+
+
+MIN_COST_FACTORIZE = 100
```

Comment:
@mloubout same story here.

In the (near) future, I'll add `devito/passes/config.py` -- a centralised place for all these parameters